real_check/test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+add_subdirectory(reals)
+add_subdirectory(both)
+set(REAL_CHECK_TEST_INC_FOLDER ${CMAKE_CURRENT_LIST_DIR}/inc CACHE INTERNAL "this is what needs to be included if using sharedLib lib" FORCE)
```

Comment:
>"this is what needs to be included if using sharedLib lib" [](http://example.com/codeflow?start=76&length=58)

Clearly copy pasted from somewhere else. Nothing with sharedLib in this file.

Reply:
Removed this line.